python/create_client/__init__.py

```diff
@@ -21,12 +21,12 @@
 
 def is_create_running(sg_connection=None):
     """
-    Launch Shotgun Create and inject the current authentication informations into the Shotgun Create session.
+    Checks for the status of Shotgun Create.
```

Comment:
```suggestion
    Check for the status of Shotgun Create.
```

We use the hum... not sure how you say this properly in english... the "non-s" ending for verbs in documentation. 😉